From ae99a2d3d3ea603402833dbc05eaa2bc2c1c2a31 Mon Sep 17 00:00:00 2001
From: Martin Basti <mbasti@redhat.com>
Date: Mon, 27 Jul 2015 16:06:39 +0200
Subject: [PATCH] Remove ico files from Makefile

Icons were removed in a4be844809179ff0a05286606df1487d81a70022 but still
persist in Makefile. This patch fixes Makefile.

https://fedorahosted.org/freeipa/ticket/823

Reviewed-By: Martin Babinsky <mbabinsk@redhat.com>
---
 install/ui/images/Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/install/ui/images/Makefile.am b/install/ui/images/Makefile.am
index 06e2cd73f4f5ab64d0c3f49d0e57d4d52dacdd9d..c77fb67d056504b03e1a0cc45b1825d22d335041 100644
--- a/install/ui/images/Makefile.am
+++ b/install/ui/images/Makefile.am
@@ -17,9 +17,7 @@ app_DATA =                              \
 	entitle-register.png		\
 	facet-tab-off.png		\
 	facet-tab-on.png		\
-	firefox-icon.png		\
 	hover-tab.png 			\
-	ie-icon.png 			\
 	ipa-banner.png			\
 	ipa-logo.png			\
 	mainnav-background.png		\
-- 
2.4.3

